modules/cnv-live-migration-limits-reftable.adoc

```diff
@@ -0,0 +1,37 @@
+
+// Module included in the following assemblies:
+//
+// cnv_users_guide/cnv-live-migration-limits.adoc
+
+[id="cnv-live-migration-limits-ref_{context}"]
+= Cluster-wide live migration limits and timeouts
+
+.Migration parameters
+|===
+|Parameter |Description |Default
```

Comment:
Default value

If you have information about acceptable ranges, consider adding another column to describe the values.